internal/tui/list.go

```diff
@@ -122,11 +127,15 @@ func (d itemDelegate) Render(w io.Writer, m list.Model, index int, listItem list
 }
 
 type ListModel struct {
-	list list.Model
+	list           list.Model
+	forgettingItem *connectionItem
```

Comment:
Can we avoid adding the `forgettingItem` field here by checking the `m.list.SelectedItem()` instead?